docs/source/getting_started.rst

```diff
@@ -0,0 +1,39 @@
+Getting Started
+===============
+
+The Data Resource Generator (DRG) is a database and API as configuration application. It is primarily used as a core component of BrightHive Data Trusts but has many :ref:`use-cases<use-cases>`. 
```

Comment:
I had a hard time making sense of this sentence. I read "is database" and then could not pice that together with API as configuration. How about this?

> The Data Resource Generator (DRG) helps you easily configure and create a RESTful API supported by a relational database without writing any code.

Reply:
The first sentence of the docs matter deeply, so we can workshop a couple solutions.

Reply:
"database as configuration" is the keyword I'm trying to catch from google searches :(

Reply:
Combined both. Let me know what you think.